spec/trips_performed.schema.json

```diff
@@ -0,0 +1,177 @@
+{
+  "primaryKey": [
+    "date",
+    "trip_id_performed"
+  ],
+  "missingValues": [
+    "NA",
+    "NaN",
+    ""
+  ],
+  "fields": [
+    {
+      "name": "date",
+      "type": "date",
+      "description": "Service date. References GTFS",
+      "constraints": {
+        "required": true
+      }
+    },
+    {
+      "name": "trip_id_performed",
+      "type": "string",
+      "description": "Identifies the trip performed.",
+      "constraints": {
+        "required": true
+      }
+    },
+    {
+      "name": "vehicle_id",
+      "type": "string",
+      "title": "ID referencing vehicles.vehicle_id",
+      "description": "Identifies the vehicle.",
+      "constraints": {
+        "required": true
+      }
+    },
+    {
+      "name": "trip_id_scheduled",
+      "type": "string",
+      "title": "ID referencing Trips.trip_id in GTFS",
+      "description": "Identifies the scheduled trip associated with the trip performed. One scheduled trip may be associated with multiple operated trips, or an operated trip may not be associated with a scheduled trip. References GTFS"
+    },
+    {
+      "name": "route_id",
+      "type": "string",
+      "title": "ID referencing routes.route_id",
+      "description": "Identifies the route. References GTFS"
+    },
+    {
+      "name": "route_type",
+      "type": "string",
+      "description": "Indicates the type of transportation used on a route. References GTFS",
+      "constraints": {
+        "enum": [
+          "Tram / Streetcar / Light rail",
+          "Subway, Metro",
+          "Rail",
+          "Bus",
+          "Ferry",
+          "Cable tram",
+          "Aerial lift",
+          "Funicular",
+          "Trolleybus",
+          "Monorail"
+        ]
+      }
+    },
+    {
+      "name": "shape_id",
+      "type": "string",
+      "title": "ID referencing shapes.shape_id",
+      "description": "Identifies a geospatial shape that describes the vehicle travel path for a trip. References GTFS"
+    },
+    {
+      "name": "direction_id",
+      "type": "string",
+      "description": "Indicates the direction of travel for a trip. References GTFS",
+      "constraints": {
+        "enum": [
+          "Outbound",
+          "Inbound"
+        ]
+      }
```

Comment:
In GTFS this is an integer enum that means travel in one direction or another (the spec includes outbound and inbound as examples). Two options:

1. (preferred) change to type: integer, enum: [0, 1] to match GTFS-static. 
2. Use strings like "Outbound / North / East / Clockwise". But, there's no guarantee that agencies group those directions consistently (someone may use direction_id 0 to mean Inbound / North / West / Clockwise).